pages/common/parquet-tools.md

```diff
@@ -0,0 +1,36 @@
+# parquet-tools
+
+> A tool to show, inspect and manipulate Parquet file.
+> More information: <https://github.com/apache/parquet-mr/tree/master/parquet-tools>.
+
+- Display the content of a Parquet file:
+
+`parquet-tools cat {{path/to/parquet}}`
+
+- Display the first few lines of a Parquet file:
+
+`parquet-tools head {{path/to/parquet}}`
+
+- Print the schema of Parquet file(s):
```

Comment:
Is this `a list of space-separated Parquet files`?
```suggestion
- Print the schema of a list of space-separated Parquet files:
```

This applies to the other `Parquet file(s)`.

Reply:
@navarroaxel Thank you for your question.

I've used the expression `file(s)` from the command's usage print.

I checked the result and understand that the expression should be `file` instead of `file(s)`
```
> parquet-tools schema ./test1.parquet ./test2.parquet
Invalid arguments: unknown extra argument "./test2.parquet"

parquet-schema:
Prints the schema of Parquet file(s)
usage: parquet-schema [option...] <input>
where option is one of:
    -d,--detailed      Show detailed information about the schema.
       --debug         Enable debug output
    -h,--help          Show this help string
       --no-color      Disable color output even if supported
    -o,--originalType  Print logical types in OriginalType representation.
where <input> is the parquet file containing the schema to show
```


I'll re-check all command using 'file(s)' expression, and push it.

Reply:
I re-checked all command whether we put multiple files to the command or not. 
I updated the document. Please check it.